test/api-test.js

```diff
@@ -0,0 +1,183 @@
+import test from 'ava';
+import sinon from 'sinon';
+import utils from './utils';
+
+global.window = utils.setupWindow();
```

Comment:
This could just be `utils.setupWindow();`, and you could modify `global.window` in the function itself.